src/server/status/plugin_status.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'lodash';
+import states from './states';
+import Status from './status';
+
+class PluginStatus extends Status {
+  constructor(plugin, server) {
+    super(`plugin:${plugin.id}@${plugin.version}`, server);
+    this.plugin = plugin;
+  }
+
+  toJSON() {
```

Comment:
Yeah, lets do that. The reason they are there is the status api, but I think it makes sense to reflect all statuses the same way, rather than differentiate based on plugin/non-plugin